doc/source/uml/h5lattice.txt

```diff
@@ -0,0 +1,37 @@
+@startuml
+class H5Lattice as "H5Lattice::Group" {
+  _v_name: String = name
+  _lattice: Table
+  _data: Table
+}
+
+class Lattice as "_table::Table" {
+  _v_name: String = "lattice"
+  attrs.type: String = type
```

Comment:
Attributes can be placed inside a group like this
The Lattice class can be described as follows

```
class H5Lattice as "H5Lattice(Group)" {
  _v_name: String = <name>
  lattice : Table = Data
  data : Table = Data
  -- Node Attributes --
  type : 
  base_vect : Double[2..3]
  size : Int[2..3]
  origin : Double[2..3]

class Data as "Data(Table)" {
  _v_name: String = <name>
  Row : DataRow
  uid : String[32]
  data: CUBARow
  mask: Bool[size of CUBARow]
}

class DataRow {
    
}
 ```